test/test.js

```diff
@@ -536,7 +536,7 @@ test('support a custom instance with custom genReqId function', function (t) {
   const dest = split(JSON.parse)
 
   let idToTest
-  function genReqId (req) {
+  function genReqId (req, res) {
```

Comment:
Please add an assertion to verify that `res` is set.

Reply:
This is done